controllers/controller-errors/errors.js

```diff
@@ -0,0 +1,15 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+  if (err.status && err.msg) 
+  res.status(err.status)
+  .send({ msg: err.msg });
+  next(err);
```

Comment:
This should be in an else statement, as in if there is status and msg, do this, else send it to the next error handler, same on the one below too